datadog_checks_base/tests/base/checks/test_agent_check.py

```diff
@@ -894,8 +895,8 @@ def test_load_configuration_models(dd_run_check, mocker):
     assert check._config_model_instance is None
     assert check._config_model_shared is None
 
-    instance_config = object()
-    shared_config = object()
+    instance_config = {}
+    shared_config = {}
```

Comment:
Just wondering, why do you change this?

Reply:
Christine changed this, but I think it's just because the previous code didn't need to iterate through them, so the mock didn't have to be specific, but now we have code that actually iterates through them